migrations/1717579194_addCookieNoticeTranslations.ts

```diff
@@ -0,0 +1,21 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  await client.items.create({
+    item_type: { type: 'item_type', id: '55443' },
+    key: 'cookie_notice_permission',
+    value: {
+      en: 'By viewing this {{ content }} you give us consent to use cookies.',
+      nl: 'Door deze {{ content }} te bekijken geef je ons toestemming om cookies te gebruiken.',
```

Comment:
This sounds like we / our website is tracking you however it's Youtube, maybe a more friendly message would be:
Playing this video requires sharing information with YouTube.

The key name should maybe also be more specific, include it's about playing a video? Thoughts on that?


**or**
We go with a generic message so we can re-use it on other content, think that's what you're going for here?
Maybe: Showing this content requires sharing information with third-party.

Reply:
We decided to go more specific as we cannot think of any other use-cases for this component in the near future.

TODO:
This sounds like we / our website is tracking you however it's Youtube, maybe a more friendly message would be:
Playing this video requires sharing information with YouTube.

The key name should maybe also be more specific, include it's about playing a video? Thoughts on that?